covid-nlp/main.ipynb

```diff
@@ -0,0 +1,3181 @@
+{
```

Comment:
<p>do we still need this code if we used the lematization preprocessing on the text?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>yes, what this code is doing is replacing the spans to their labels, so for example if we had novel covid, it will be changed to the label, what is meant by Lemma matches is the matched spans for the rgx rules.</p>